src/main/java/storagesystem/viewcontroller/settings/TeamPageController.java

```diff
@@ -204,17 +214,17 @@ private void removeMemberButtonPressed() {
     private void removeMemberFromTeam(int userID) {
         boolean memberFound = false;
         //remove member from team
-        for (int i : currentlySelectedTeam.getAllMemberIDs()) {
+        for (int i : StoreIT.getCurrentTeam().getAllMemberIDs()) {
```

Comment:
can be  changed to
`if (StoreIT.getCurrentTeam().getAllMemberIDs().contains(userID)) {`